core/src/main/java/org/geysermc/geyser/translator/inventory/item/nbt/EnchantmentTranslator.java

```diff
@@ -70,6 +70,20 @@ public void translateToBedrock(GeyserSession session, CompoundTag itemTag, ItemM
         if (!newTags.isEmpty()) {
             itemTag.put(new ListTag("ench", newTags));
         }
+
+        if (sweepingEdge > 0) {
+            CompoundTag displayTag = itemTag.get("display");
+            if (displayTag == null){
+                itemTag.put(new CompoundTag("display"));
+                displayTag = itemTag.get("display");
+            }
+            ListTag loreTag = displayTag.get("Lore");
+            if (loreTag == null){
+                displayTag.put(new ListTag("Lore"));
+                loreTag = displayTag.get("Lore");
+            }
+            loreTag.add(new StringTag("", "§7Sweeping Edge " + levelToRomanNumeral(sweepingEdge)));
```

Comment:
Instead of directly referencing the section symbol, please use ChatColor.

Reply:
Also: there should be translation strings for both roman numerals and Sweeping Edge. Please use these instead. (Dig up the old enchantment GUI made from a hopper if you want an example)

Reply:
addressed, thanks for the pointer